lib/smart_city/dataset.ex

```diff
@@ -41,8 +40,6 @@ defmodule SmartCity.Dataset do
       "authUrl": "",
       "cadence": "",
       "dataName": "",        // ~r/[a-zA-Z_]+$/
-      "orgId": "",
-      "orgName": "",         // ~r/[a-zA-Z_]+$/
```

Comment:
This is probably the answer to my question "why was the orgName removed" in the smart_city_test repo

Reply:
yes, indeed! It was removed because it is no longer available from the dataset. It can only be found in the organization.